public/app/features/explore/Table/TableContainer.tsx

```diff
@@ -51,9 +52,11 @@ export class TableContainer extends PureComponent<Props> {
   }
 
   getTableTitle(dataFrames: DataFrame[] | null, data: DataFrame, i: number) {
-    let title = data.name ? `Table - ${data.name}` : 'Table';
+    let title = data.name
+      ? t('explore.table.title-with-name', 'Table - {{name}}', { name: data.name })
+      : t('explore.table.title', 'Table');
     if (dataFrames && dataFrames.length > 1) {
-      title = `Table - ${data.name || data.refId || i}`;
+      title = `${t('explore.table.title-with-dataframes', 'Table')} - ${data.name || data.refId || i}`;
```

Comment:
Like mention before, we should not manually concatenate translation strings. This should be a parameterised phrase like `explore.table.title-with-name` above it.

However, I've refactored this logic to not require the duplicate phrases.